ak_vendor/templates/report_template.html

```diff
@@ -526,6 +526,10 @@
       margin-bottom: 0.5em;
     }
 
+    .regulatory__table__pcidss4__title {
+      margin-bottom: 0.5em;
+    }
```

Comment:
I don't see this css class being used anywhere in this HTML file. Am I missing something here?